docs/question.md

```diff
@@ -344,14 +334,37 @@ That question would be rendered like this:
 <p>This is some <strong>Markdown</strong> text.</p>
 ```
 
+!!! warning
+
+    It is reccommended to not indent the text inside markdown blocks, as markdown recognizes indentation as a code block.
```

Comment:
Besides the typo (recommended, not reccommented), this sounds weird...
```suggestion
    Note that markdown recognizes indentation as a code block, so text inside these tags should not be indented with the corresponding HTML content.
```

Reply:
Oops, good catch. Thanks!